README.md

```diff
@@ -37,8 +37,8 @@ then...
 
 Make sure you have a reasonably recent version of Node (7.6+ minimum). You can check your version of node by running
 
-```
-node -v
+```sh
+$ node -v
```

Comment:
So I'm actually wanting to remove all $ from shell commands in documentation, because we've found they're confusing to beginners.

Reply:
Hi @jamonholmgren ! 
No worries, how can we make the difference between the command and the output clear, then? 🤔 
For example, in `CONTRIBUTING.md`, there is : 
```sh
$ ignite --version
<current version here>
$ which ignite
/usr/local/bin/ignite
$ ignite new UberForHeadLice
...
```
I feel the following is confusing :
```sh
ignite --version
<current version here>
which ignite
/usr/local/bin/ignite
ignite new UberForHeadLice
...
```

Reply:
You're right. Maybe we should leave the $ then.